.travis.yml

```diff
@@ -8,28 +8,40 @@ matrix:
           sudo: required
           group: deprecated-2017Q3
           go: 1.8
-          env:
-            secure: "Br0fD9CAKm8gqyEuwmltNJd4dGJCxPpj6feugHlO+CFFwfE/+kJKkpTlsDuRfrUzUDlWiETNPf0XGSjvPFqZExnLCE9XQh2+XF6u+S3YBWfM+rbbyRVAK6BlTwmt0u3jRJ2JP7spedTGZA+qfIWI+UkwoOexo7NcqtMPLahiZzheaaad8y3J+crHQCrB/kPrhLqKVMEOkIbveFdfV2QLfCOWgqP8e1LGZhPZ2N4QcNo0iB5uI4ZyYszTZDniXFKxz7kBs4tl4ZQDqRHqL02qKPsjbvjrZp83ql+PbC2dpgXi9YpaDuBEqKKX1rTQP5ppcwbobot5U3ItHWzpXbLCdsWxvbde/0enjMmOF1wwl71hPYIf7PkQmNAWXRtL2Z1TguO/dKCeXBLDER4YDQ79GYpikAMqnrRLou4rsyZrNUzg8aHbELzAHppDSpqEJN8ymGgWgmWBT8yPaWVwN5CjPFOxLPrVEObcwcNSGOcOvtmUGCnXKSZminNZUjz5QfWqpA+LYCUIbeOI2X2D+iQWklWONaU2A+PcNLaaegwwj4K+9ZlU08Ed8Ud5ZGhjN9s72OtoFPdSHAyBgf/qxIdnTzmmE+SQ90mDtG3VqjHy28Ix7cIGqaIEC8shFb0kKHqQ7AVkLPm0bsh2fbbYu/5YVpXHfxlKWfOLBV14bHUq3v0="
           services: docker
 
+env:
+  global:
+      - zip_file_name=OpenWhisk_CLI
+  secure: "Br0fD9CAKm8gqyEuwmltNJd4dGJCxPpj6feugHlO+CFFwfE/+kJKkpTlsDuRfrUzUDlWiETNPf0XGSjvPFqZExnLCE9XQh2+XF6u+S3YBWfM+rbbyRVAK6BlTwmt0u3jRJ2JP7spedTGZA+qfIWI+UkwoOexo7NcqtMPLahiZzheaaad8y3J+crHQCrB/kPrhLqKVMEOkIbveFdfV2QLfCOWgqP8e1LGZhPZ2N4QcNo0iB5uI4ZyYszTZDniXFKxz7kBs4tl4ZQDqRHqL02qKPsjbvjrZp83ql+PbC2dpgXi9YpaDuBEqKKX1rTQP5ppcwbobot5U3ItHWzpXbLCdsWxvbde/0enjMmOF1wwl71hPYIf7PkQmNAWXRtL2Z1TguO/dKCeXBLDER4YDQ79GYpikAMqnrRLou4rsyZrNUzg8aHbELzAHppDSpqEJN8ymGgWgmWBT8yPaWVwN5CjPFOxLPrVEObcwcNSGOcOvtmUGCnXKSZminNZUjz5QfWqpA+LYCUIbeOI2X2D+iQWklWONaU2A+PcNLaaegwwj4K+9ZlU08Ed8Ud5ZGhjN9s72OtoFPdSHAyBgf/qxIdnTzmmE+SQ90mDtG3VqjHy28Ix7cIGqaIEC8shFb0kKHqQ7AVkLPm0bsh2fbbYu/5YVpXHfxlKWfOLBV14bHUq3v0="
+
 git:
   depth: 3
 
 before_install:
   - pip install --upgrade pip setuptools
+  - go get github.com/golang/lint/golint
 
 install:
-  - ./tools/travis/setupscan.sh
   - export DEPLOY_BUILD_READY=false
-  - go get -u github.com/golang/lint/golint
-  - go get -u github.com/stretchr/testify
-  - go get -u github.com/spf13/viper
+
+before_script:
+  # Clone needed git repositories for the build -- TODO: Pick a commit?
+  - cd $TRAVIS_BUILD_DIR/..
+  - git clone --depth 3 https://github.com/apache/incubator-openwhisk.git
+  - export OPENWHISK_HOME="$(cd '../incubator-openwhisk' && pwd )"
+  - git clone --depth 3 https://github.com/apache/incubator-openwhisk-utilities.git
+  # These should be temporary until next release of gogradle
+  - git clone --depth 3 https://github.com/jonpspri/gogradle.git
+  - cd gogradle && ./gradlew --console=plain assemble
 
 script:
+  - cd $TRAVIS_BUILD_DIR
+  - ./gradlew --console=plain buildBinaries release -PcrossCompile=true -PzipFileName=${zip_file_name}
```

Comment:
test_openwhisk.sh builds again, is it needed there?

Reply:
I think the build was removed from test_openwhisk.sh (see my comments above).  In fact, this should be the only build that happens in the Travis processing.